src/cmd/singularity/cli/verify.go

```diff
@@ -17,19 +17,20 @@ import (
 
 func init() {
 	VerifyCmd.Flags().SetInterspersed(false)
+	VerifyCmd.Flags().StringVarP(&keyServerURL, "url", "u", defaultKeysServer, "specify the key server URL")
 	SingularityCmd.AddCommand(VerifyCmd)
 }
 
 // VerifyCmd singularity verify
 var VerifyCmd = &cobra.Command{
 	DisableFlagsInUseLine: true,
-	Args:   cobra.ExactArgs(1),
+	Args:   cobra.RangeArgs(1, 2),
```

Comment:
I think this should be `cobra.ExactArgs(1)`, as `verify` only takes one positional argument.